src/content_scripts/autocorrect.js

```diff
@@ -255,6 +257,7 @@ function firstDifferenceIndex(a, b) {
  */
 function autocorrect(event) {
     // console.log('keydown', event.key, event.key.length, event.keyCode);
+    // Exclude all keys that do not produce a single Unicode character
     if (!((event.key.length === 0 || event.key.length === 1 || event.keyCode === 13 || event.key === "Unidentified") && !event.ctrlKey && !event.metaKey && !event.altKey)) {
```

Comment:
Ahh wait, new problem: ` keyCode`  is deprecated…
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Though you cannot use `key`  for non-printable characters, can you?

But they recommend the `keypress`  event… :thinking:

Reply:
Yes, unfortunately from my testing `key` does not return consistent values for non-printable characters, so I had to use  `keyCode`. Note that the keypress event is also deprecated: https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event.